README.md

```diff
@@ -86,4 +86,18 @@ You have access to all of the standard Mongoose error message templating:
 
 *   `{PATH}`
 *   `{VALUE}`
-*   `{TYPE}`
\ No newline at end of file
+*   `{TYPE}`
+
+
+Case Insensitive
+---------------------
+
+You can add `case insensitive` option to the key. Then `john.smith@gmail.com` and `John.Smith@gmail.com` will be treated as duplicates.
+
+```
```

Comment:
Can you include the code language so we're consistent with other code segments in the readme. So ```js